startup.m

```diff
@@ -0,0 +1,15 @@
+
+% First written by : Gordon G. Parker, Michigan Technological University
+% Modifed for WEC-Sim by: Sal Husain, sal.husain@nrel.gov
+% Last modified : June 9th, 2021
+
+
+% Root directory of this running .m file
+projectRootDir = fileparts(mfilename('fullpath'));

```

Comment:
@salhus Thank you for cleaning up and resubmitting your pull request. 

In this file you create the `projectRootDir` variable which remains after the file executes while in the cleanup.m line 17 deletes this variable. I do not see any additional dependency on this path so can we also add the `clear` statement in the startup.m?

Reply:
Yes I agree that can be cleared up. 
I usually use that variable in my scripts when I need to save variables programmatically but it is not needed per se.